content/blog/open-sourcing-workshops-on-demand-part2-deploying-the-backend.md

```diff
@@ -0,0 +1,389 @@
+* Commit and push changes to your repo
+* Create an `install.priv` file located in `install` directory when using a private repo (consider looking at [install.repo](https://github.com/Workshops-on-Demand/wod-backend/blob/main/install/install.repo) file for better understanding of the variables.
+
+  * Define the WODPRIVREPO and WODPRIVBRANCH variables as follows:
+
+    * WODPRIVBRANCH="main"
+    * WODPRIVREPO="git@github.com:Account/Private-Repo.git wod-private"
+
+**Note: When using a token**
+
+Please refer to the following [url](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) to generate a `token` file in `install` directory of wod-backend:
+
+* edit the `install.priv` file located in `install` directory of wod-backend:
+
+  * create line before variable declaration: ``token=`cat $EXEPATH/token` ``
+  * use the token in the url WODPRIVREPO="git clone https://user:$token@github.com/Account/wod-private.git wod-private"
+
+Y﻿ou are now ready to perform the installation again to support a private repository. 
+
+```shellsession
+cd $HOME/wod-backend/install
+sudo ./install.sh -t backend -g staging -b jup.example.net -f notebooks.example.io -a api.example.io -e notebooks.example.io -s sender@example.io
+```
+
+Please note that this setup phase can be conccurent with the public setup phase. Indeed, the install script should detect the presence of the private repository owing to the presence of the install.priv file. It will automatically adjust the different scripts and variables to add the relevant content. It will actually overload some of the variables with private ones.
+
+Y﻿ou have now a working Workshops-on-Demand backend server in place. Congratulations! The next article in the series will help you better understand the lifecycle of the backend server. How does a workshop registration works from the backend server 's side? How do you manage this server on a daily basis? How and when do you need to update it ? All these questions will be answered in the next article. And from there, we will move to the frontend side of things and finally to the workshop's creation process.
+
+Please be sure to drop back at [HPE DEV](https://developer.hpe.com/blog) for a follow up on this. Check out also the Hack Shack for new [workshops](https://developer.hpe.com/hackshack/workshops)! [Data Visualization 101](https://developer.hpe.com/hackshack/replays/42) is now available! HPE GreenLake for Compute Operations Management API 101 on its way too. Stay tuned!
```

Comment:
Change "drop back" to "check back". Change "HPE DEV" to "the "HPE Developer blog site" and change "for a follow up on this" to "to read all the articles in this series."

The second sentence also needs to change to: "Also, be sure to check out the Hack Shack for new workshops. Data Visualization 101 is now available!